docs/topics/db/queries.txt

```diff
@@ -669,6 +669,20 @@ The ``F()`` objects support bitwise operations by ``.bitand()``, ``.bitor()``,
 
     Support for ``.bitxor()`` was added.
 
+.. _lookups-in-expressions:
+
+Lookups in expressions
+----------------------
+
+.. versionadded: 3.2
+
+Django also supports using lookups in expressions.
+
+This example finds all ``Entry`` objects published in the same year as they
+were last modified:
+
+    >>> Entry.objects.filter(pub_date__year=F('mod_date__year'))
```

Comment:
I feel like this probably should cover more ground, but I'm not sure what exactly to add.

Reply:
I would at least add examples for aggregations and `OuterRef()`.